R/autoplot.R

```diff
@@ -0,0 +1,156 @@
+#' Automatically plot an epi_df
+#'
+#' @param object An `epi_df`
+#' @param ... <[`tidy-select`][dplyr_tidy_select]> One or more unquoted
+#'   expressions separated by commas. Variable names can be used as if they
+#'   were positions in the data frame, so expressions like `x:y` can
+#'   be used to select a range of variables.
+#' @param .color_by Which variables should determine the color(s) used to plot
+#'   lines. Options include:
+#'   * `all_keys` - the default uses the interaction of any key variables
+#'     including the `geo_value`
+#'   * `geo_value` - `geo_value` only
+#'   * `other_keys` - any available keys that are not `geo_value`
+#'   * `.response` - the numeric variables (same as the y-axis)
+#'   * `all` - uses the interaction of all keys and numeric variables
+#'   * `none` - no coloring aesthetic is applied
+#' @param .facet_by Similar to `.color_by` except that the default is to display
+#'   each numeric variable on a separate facet
+#' @param .base_color Lines will be shown with this color. For example, with a
+#'   single numeric variable and faceting by `geo_value`, all locations would
+#'   share the same color line.
```

Comment:
question: Is this the correct usage?

```
autoplot(jhu_csse_daily_subset, case_rate_7d_av, .base_color = "red", .facet_by = "geo_value")
```

issue: The line color isn't changing when I do this, or if I try a hex color code.

Reply:
No. The default colouring is controlled by `.color_by`. So you'd need:
```r
autoplot(jhu_csse_daily_subset, case_rate_7d_av, .color_by = "none", .base_color = "red", .facet_by = "geo_value")
```

Reply:
Added this as an example.